exporter/opentelemetry-exporter-jaeger-proto-grpc/tests/test_jaeger_exporter_protobuf.py

```diff
@@ -189,7 +189,7 @@ def test_translate_to_jaeger(self):
                 parent=None,
                 resource=Resource({}),
                 instrumentation_info=InstrumentationInfo(
-                    name="name", version="version"
+                    name="name", version="version", schema_url="schema_url"
```

Comment:
are the changes to the tests necessary if this is an optional parameter that doesn't appear to be tested anywhere?

Reply:
Thank you for pointing it out, I have modified it accordingly.